Galaxy.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>Galaxy</title>
+		<link rel="stylesheet" href="visual.css">
+		<link rel="shortcut icon" href="icon.ico" type="image/x-icon">
+	</head>
+	<body>
+	<header class="header">
+		
+		<nav class="bar">
+			<a href="#">Home</a>
+			<a href="#">About us</a>
+			<a href="#">Charity</a>
+			<a href="#">Contacts</a>
+			<p>UNKNOWN UNIVERSE</p>
+		</nav>
+		
+	</header>
+		
+		<div class="whole-universe">
+			<ul class="voating">
+					<li><input type="radio" name="planet" value="1"><i>Uranus</i></li>
+					<li><input type="radio" name="planet" value="2"><i>Mars</i></li>
+					<li><input type="radio" name="planet" value="3"><i>Io</i></li>
+					<li><input type="radio" name="planet" value="4"><i>Phobos</i></li>
+					<li><input type="radio" name="planet" value="5"><i>GJ 273 b</i></li>
+					<li><p class="voting1"><input type="submit" value="Choose your destiny"></p></li>
+					<h3>«That's one small step for a man, one giant leap for mankind».(с)Neil Armstrong</h3>
+			</ul>	  
+			<div class="content">
```

Comment:
This one I see as `section` tag https://developer.mozilla.org/ru/docs/Web/HTML/Element/section made by `articles` for each portion of data

Reply:
try like <article> which wraps <a> + <p> + <img>
https://developer.mozilla.org/ru/docs/Web/HTML/Element/article

Reply:
ok, changes will be added